### PR TITLE
chore: bump dynamic-credential-provider to v0.5.3

### PR DESCRIPTION
**Pull request by supershal**
**What problem does this PR solve?**:
bumps dynamic credentials provider to v0.5.3

### CODE REVIEW

File: ansible/group_vars/all/defaults.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ aws_images: []
 
 extra_images: []
 
-containerd_supplementary_images: ["ghcr.io/mesosphere/toml-merge:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.5.0"]
+containerd_supplementary_images: ["ghcr.io/mesosphere/toml-merge:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.5.3"]
```
Comment (jimmidyson) [inline ansible/group_vars/all/defaults.yaml:62]: Why do we need both `ghcr.io/mesosphere/dynamic-credential-provider:v0.2.0` and `v0.5.3`?

Comment (supershal) [reply ansible/group_vars/all/defaults.yaml:62]: I honestly does not have context about this. @dkoshkin any idea?

Comment (dkoshkin) [reply ansible/group_vars/all/defaults.yaml:62]: This was the PR that added `v0.5.0` https://github.com/mesosphere/konvoy-image-builder/pull/1019
Konvoy is still using `v0.2.0` and the old API https://github.com/mesosphere/konvoy2/blob/051fdf94628a09028d0c4befe8d27b04876d37f7/pkg/capi/generator/templates/dynamic-credential-provider-config.yaml.gotmpl#L12

I think we should keep `v0.2.0` like you did here and cut another ticket to do the API migration in Konvoy.

**Review by dkoshkin — APPROVED** [review]

File: ansible/group_vars/all/defaults.yaml (hunk 1 above)
Comment (supershal) [reply ansible/group_vars/all/defaults.yaml:62]: Thank you for the context

**Pull request merged**